.travis.yml

```diff
@@ -32,6 +32,7 @@ install:
       export LD_LIBRARY_PATH="$HOME/miniconda/envs/chainercv/lib:$LD_LIBRARY_PATH";
       conda env create -f environment.yml;
       source activate chainercv;
+      (cd ..; pip install -e 'git+https://github.com/pdollar/coco.git#egg=pycocotools&subdirectory=PythonAPI')
```

Comment:
Do we need `cd`?

Reply:
I didn't want to have anything under `chainercv/`.

Reply:
Does it download something into current directory?

Reply:
Yes.